pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -203,7 +208,12 @@ let
   # Flag change between versions (e.g. "--enable-armvfp" -> "--enable-vfp" changed in v1.1)
   chgFlg = chgVer: oldFlag: newFlag: if reqMin chgVer then newFlag else oldFlag;
 
+  # Disable dependency that needs fixes before it will work on Darwin
+  disDarwinFix = origArg: if stdenv.isDarwin then false else origArg;
+
   isCygwin = stdenv.isCygwin;
+  isDarwin = stdenv.isDarwin;
+  isLinux = stdenv.isLinux;
```

Comment:
For future reference: in ``let`` you can instead have ``inherit (stdenv) isCygwin isDarwin isLinux;``.